Week3/homework/js-exercises/Project_CreditCardValidator.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+function validNumber(num) {
+    
+    const sumDigits = str => { //This function sums the digits of the credit card number in the string. Below I used an if statement to make sure that sum of the digits is greater than 16.
+        let sum = 0;
+        for (let char of str) {
+          sum += +char;
+        }
+        return sum
+      };
```

Comment:
Why not use `reduce` here as well 🙂

Reply:
I kind of got lost with reduce :). I have to check reduce again.